articles/azure-monitor/app/opencensus-python-request.md

```diff
@@ -41,6 +42,8 @@ First, instrument your Python application with latest [OpenCensus Python SDK](./
     }
     ```
 
+You can find an example Django application in the sample Azure Monitor OpenCensus Python samples repository located [here](https://github.com/Azure-Samples/azure-monitor-opencensus-python/tree/master/azure_monitor).
```

Comment:
Please update these links to point to the READMEs of the respective samples once those get merged in the samples repo.

Reply:
Done.